crates/core/prelude/text-encoding.js

```diff
@@ -4,6 +4,7 @@
 
     class TextDecoder {
         constructor(label = "utf-8", options = {}) {
+            label = label.trim().toLowerCase();
```

Comment:
Yes but it's not necessary to make the `textencoder_constructor-non-utf` suite pass. Merging #192 will take care of it anyway since this PR and #192 are both updating the TextEncoder constructor.